Project2015To2017/Transforms/NuGetPackageTransformation.cs

```diff
@@ -76,10 +77,8 @@ private string PopulatePlaceHolder(string placeHolder, string nuSpecValue, strin
 			{
 				return assemblyAttributeValue ?? nuSpecValue;
 			}
-			else
-			{
-				return nuSpecValue;
-			}
+
+			return nuSpecValue;
```

Comment:
I guess we have different styles here. This is one resharper warning we disable at work because we prefer the clarity of if/else